aerosol_dynamics/particle.py

```diff
@@ -91,72 +101,114 @@ def reduced_mass(self, other) -> float:
         The reduced mass is an "effective inertial" mass that allows a two body
         problem to be solved as a one body problem.
         """
+
         return self.mass() * other.mass() / (self.mass() + other.mass())
 
     def reduced_friction_factor(self, other) -> float:
         """Returns the reduced friction factor between two particles. [N-s/m]
-       
         Similar to the reduced mass, the reduced friction factor allows a two
         body problem to be solved as a one body problem.
         """
-        return (self.friction_factor() * other.friction_factor()
-            / (self.friction_factor() + other.friction_factor()))
+
+        return (
+            self.friction_factor() * other.friction_factor()
+            / (self.friction_factor() + other.friction_factor())
+        )
 
     def coulomb_potential_ratio(self, other) -> float:
         """
+        Calculates the Coulomb potential ratio.
         """
+
         numerator = -1 * self.charge() * other.charge() * (pp.ELEMENTARY_CHARGE_VALUE ** 2)
         denominator = 4 * np.pi * pp.ELECTRIC_PERMITTIVITY * (self.radius() + other.radius())
         return numerator / (denominator * pp.BOLTZMANN_CONSTANT * pp.TEMPERATURE)
 
     def coulomb_enhancement_kinetic_limit(self, other) -> float:
         """
+        Calculates the Coulomb enhancement for a particle-particle interaction
         """
+
         coulomb_potential_ratio = self.coulomb_potential_ratio(other)
-        return 1 + coulomb_potential_ratio if coulomb_potential_ratio >=0 else np.exp(coulomb_potential_ratio)
+        return 1 + coulomb_potential_ratio if coulomb_potential_ratio >= 0 \
+            else np.exp(coulomb_potential_ratio)
 
     def coulomb_enhancement_continuum_limit(self, other) -> float:
         """
+        Calculates the Coulomb enhancement for a particle-particle interaction
         """
+
         coulomb_potential_ratio = self.coulomb_potential_ratio(other)
-        return coulomb_potential_ratio / (1 - np.exp(-1*coulomb_potential_ratio)) if coulomb_potential_ratio != 0 else 1
+        return coulomb_potential_ratio / (
+            1 - np.exp(-1*coulomb_potential_ratio)
+        ) if coulomb_potential_ratio != 0 else 1
 
     def diffusive_knudsen_number(self, other) -> float:
         """
+        Calculates the diffusive Knudsen number for a particle-particle interaction
+        TODO:
+        - do return statements in steps -> easier to debug
```

Comment:
I'm for separating this one into two or more statements. 

One consideration to keep in mind is between "easier to debug" and "uses fewer resources by not having to store values".

Reply:
That's a good point!

Reply:
I added that as another TODO item. Will revist.